tcf_website/templates/course/course_professor.html

```diff
@@ -164,24 +184,33 @@ <h3 class="font-weight-light text-tcf-orange">
         </div>
 
         <div class="reviews-container mb-5 px-3">
-            <div class="reviews-toolbar d-flex">
-                <div class="ml-auto dropdown btn-group">
-                    <button class="btn bg-white" disabled>Sort by</button>
-                    <button id="review-sort-select" class="btn btn-primary dropdown-toggle" type="button" id="sortOptions"
-                            data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"></button>
-                    <div class="dropdown-menu" aria-labelledby="sortOptions">
-                        <button id="votes-sort-btn" onclick="sortReviews(this.id)" class="dropdown-item" type="button">
-                            Most Helpful
-                        </button>
-                        <button id="recent-sort-btn" onclick="sortReviews(this.id)" class="dropdown-item" type="button">
-                            Most Recent
-                        </button>
-                        <button id="highrating-sort-btn" onclick="sortReviews(this.id)" class="dropdown-item" type="button">
-                            Highest Rating
-                        </button>
-                        <button id="lowrating-sort-btn" onclick="sortReviews(this.id)" class="dropdown-item" type="button">
-                            Lowest Rating
-                        </button>
+            <div class="reviews-toolbar d-flex flex-md-row flex-column justify-content-left justify-content-md-center">
+                <div class="reviews-header d-inline-flex">
+                    <h3 class="h2">{{ reviews|length }} Comments</h3>
+
+                </div>
+                <div class="ml-auto">
```

Comment:
I think this could look good to the right of `X comments`, kind of like "3 Reviews [ + Add yours! ]

Reply:
mm I'll play around with the flexbox again